config/local.envvars

```diff
@@ -29,3 +29,6 @@ CONFIG_READONLY=0
 SIMPLEI_ENV_COLOUR=#637C16
 SIMPLEI_ENV_NAME=Local
 GOOGLE_MAP_API_KEY=AIzaSyBBRzCpU4iW0N6xRe4dVhrf0zqiye31hik
```

Comment:
While it's here, reckon we should clear `GOOGLE_MAP_API_KEY ` out as well? That'd be a clear nudge to the developer to source or set up a new key.

Reply:
Yup, makes sense to.